web/html/ParamsApp/php/InsertParamValues.php

```diff
@@ -21,57 +21,66 @@
 
 
 
-// set the $_POST variable (not sure why)
-if (($_SERVER["REQUEST_METHOD"] == "POST") /* && empty($_POST) */) {
-	if (isset($_SERVER["CONTENT_TYPE"]) && (strpos($_SERVER["CONTENT_TYPE"], "application/json") !== false)) {
-		$_POST = json_decode(file_get_contents('php://input'), true);
-	}
-}
-
-
 // Get the type, possible min, and possible max values for this item in the DB
-$name = $_POST['newParamValues'][0]['name'];
-$user_input = $_POST['newParamValues'][0]['Value'];
-$limitation_query = "SELECT type, min, max FROM $AbstractParameterTable WHERE name='$name'";
-$limitation_result = $conn->query($limitation_query);
-if ($limitation_result === false) {
-	writeError("1", "102", "Limitation Query failed", "");
-    exit();
+foreach ($data['newParamValues'] as $newParam) {
+    $name = $newParam['name'];
+    $user_input = $newParam['Value'];
+    $limitation_query = "SELECT type, min, max FROM $AbstractParameterTable WHERE name='$name'";
```

Comment:
Does this query apply to all params - i.e., do all params have min/max limits?

Reply:
figured out all params except for params of type "string" have min/max limits. However, `is_user_input_valid()` accounts for this